pillar/apps/odlvideo.sls

```diff
@@ -141,6 +141,7 @@ django:
   states:
     setup:
       - apps.odlvideo.install
+      - apps.odlvideo.configure
```

Comment:
This state should go under the path `states:config` so that it gets run when the `django.config` state gets executed. You can refer to the mitx-cas or starcellbio pillar for an example.